src/uct/ib/base/ib_iface.c

```diff
@@ -775,6 +836,41 @@ static ucs_status_t uct_ib_iface_set_moderation(struct ibv_cq *cq,
     return UCS_OK;
 }
 
+static ucs_status_t uct_ib_device_set_gid_info(uct_ib_device_t *dev,
```

Comment:
rename to uct_ib_iface_init_gid_info(iface, config) and move to ib_iface.c
it will simplify the code somewhat

Reply:
it's already in ib_iface.c... i'll rename